src/app/ui/timeline.cpp

```diff
@@ -88,6 +88,8 @@ enum {
   PART_FRAME_TAG,
 };
 
+static const int base_size = 12;
```

Comment:
Here we can add a `// TODO move this to theme dimensions` or include a patch moving this as a Timeline static member (`Timeline::m_baseSize`?) and initialize it with `m_baseSize = skinTheme()->dimensions.timelineBaseSize()` (adding this `timeline_base_size` element in `skin.xml` at the `<dimensions>`)